back-end/src/test/resources/application-test.yml

```
@@ -11,11 +11,17 @@ spring:
+      api-key: dummy-key
+      chat:
+        options:
+          model: GPT-4o-mini
+      base-url: https://api.openai.com
 
 server:
   http.port: 80
   port: 443
-
 cors:
   allowed-origins: "http:localhost"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)